### PR TITLE
Adding header_html and footer_html fields in company #7926

### PR DESCRIPTION
**Pull request by priyankajain18**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -36,20 +42,73 @@ def run(self):
...
+
+
+config = ConfigParser.ConfigParser()
+config.readfp(open('tryton.cfg'))
+info = dict(config.items('tryton'))
+print info
+for key in ('depends', 'extras_depend', 'xml'):
+    if key in info:
+        info[key] = info[key].strip().splitlines()
 VERSION = '3.4.0.4'
```
Comment (aroraumang) [inline setup.py]: Remove this

File: company.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+from trytond.model import fields
+
+__metaclass__ = PoolMeta
+__all__ = ['Company']
+
+
+class Company:
+    __name__ = 'company.company'
+
+    header_html = fields.Text('Header Html')
```
Comment (prakashpp) [inline company.py]: why cant we reuse company header and footer?

File: view/company_form.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<data>
+    <xpath expr="/form/notebook/page[@id='reports']/field[@name='footer']" position="after">
```
Comment (prakashpp) [inline view/company_form.xml]: create a new page `HTML Reports`

File: setup.py [hunk 4]
```diff
@@ -36,20 +42,72 @@ def run(self):
...
+
+        if test_result.wasSuccessful():
+            sys.exit(0)
+        sys.exit(-1)
+
+
+config = ConfigParser.ConfigParser()
+config.readfp(open('tryton.cfg'))
+info = dict(config.items('tryton'))
+print info
```
Comment (prakashpp) [inline setup.py]: remove

**Pull request merged**